src/settingsmanager.cpp

```diff
@@ -62,6 +68,9 @@ void SettingsManager::setKiwixServerPort(int port)
 
 void SettingsManager::setZoomFactor(qreal zoomFactor)
 {
+    if (zoomFactor > 1)
+        zoomFactor /= 100;
+
```

Comment:
This is surprising. If the user enter a value of `1.5`, we will set 1.5%

Reply:
That's because in the UI part, QSpinBox takes value from 0 to 100, so I need to divide it to the intended value. The `if` here is in case other parts use this function with correct value (0-1)